source/includes/stackpath/_origin_settings.md

```diff
@@ -0,0 +1,192 @@
+`origin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
+`origin.path` <br/> *string* | The origin's HTTP request path.
+`origin.hostname` <br/> *string* | The origin's HTTP request hostname. Can be a valid IPv4 address or a valid domain name. 
+`origin.port` <br/> *int* | The origin's HTTP port. Must be one of [80, 8080, 443, 1935, 9091].
+`origin.securePort` <br/> *int* | The origin's HTTPS port. Must be one of [80, 8080, 443, 1935, 9091].
+`origin.priority` <br/> *string* | The origin's priority to the scope. Must be one of ["ZERO", "ONE"]. "ZERO" has the highest priority.  
+`backupOriginEnabled`<br/>*boolean* | Specifies if a backup origin for the site is configured. To configure backupOrigin's settings, this property must be set to true. 
+`backupOriginExcludeCodes` <br/>*string* | Requests are made to the backup origin on any 4xx or 5xx response codes returned from the primary origin. This property specifies the response status codes for which calls to the backup origin must not be made. Multiple response codes can be excluded. e.g: "410, 411, 412".  Asterisks can be used to cover a range of codes. e.g. All the 4xx codes can be covered using "4*". 
+`backupOrigin` <br/> *Object* | The secondary origin that the CDN uses to pull content from when the primary origin is not available. 
+`backupOrigin.id` <br/> *UUID* | A backup origin's unique identifier. 
+`backupOrigin.stackId` <br/> *UUID*  | The ID of the stack that a site belongs to.
+`backupOrigin.siteId` <br/> *UUID*  | A site's unique identifier. 
+`backupOrigin.dedicated` <br/> *boolean* | Specifies if a backup origin is dedicated to a CDN site. 
+`backupOrigin.authenticationEnabled` <br/> *boolean* | Specifies if the backup origin is using basic http authentication. To update backupOrigin.username and backupOrigin.password, this property must be set to true.
+`backupOrigin.username` <br/> *string* | Username to use when authenticating with the backup origin. 
+`backupOrigin.password` <br/> *string* | Password to use when authenticating with the backup origin. 
+`backupOrigin.sslValidationEnabled` <br/> *boolean* | Specifies if SSL validation for the backup origin is enabled. 
+`backupOrigin.commonCertificateName` <br/> *string* | Common name to validate SSL origin requests against.
+`backupOrigin.path` <br/> *string* | The backup origin's HTTP request path.
+`backupOrigin.hostname` <br/> *string* | The backup origin's HTTP request hostname. Can be a valid IPv4 address or a valid domain name. 
+`backupOrigin.port` <br/> *int* | The backup origin's HTTP port. Must be one of [80, 8080, 443, 1935, 9091].
+`backupOrigin.securePort` <br/> *int* | The backup origin's HTTPS port. Must be one of [80, 8080, 443, 1935, 9091].
+`backupOrigin.priority` <br/> *string* | The backup origin's priority to the scope. Must be one of ["ZERO", "ONE"]. "ZERO" has the highest priority.  
```

Comment:
Does setting `backupOrigin.priority` to `ONE` replace the `origin`? If so, maybe we can be more explicit about that.

Reply:
9199096